server/tinylicious/package.json

```diff
@@ -25,17 +25,17 @@
   },
   "dependencies": {
     "@fluidframework/common-utils": "^0.24.0-0",
-    "@fluidframework/gitresources": "^0.1014.0-0",
-    "@fluidframework/protocol-base": "^0.1014.0-0",
-    "@fluidframework/protocol-definitions": "^0.1014.0-0",
-    "@fluidframework/server-lambdas": "^0.1014.0-0",
-    "@fluidframework/server-local-server": "^0.1014.0-0",
-    "@fluidframework/server-memory-orderer": "^0.1014.0-0",
-    "@fluidframework/server-services-client": "^0.1014.0-0",
-    "@fluidframework/server-services-core": "^0.1014.0-0",
-    "@fluidframework/server-services-shared": "^0.1014.0-0",
-    "@fluidframework/server-services-utils": "^0.1014.0-0",
-    "@fluidframework/server-test-utils": "^0.1014.0-0",
+    "@fluidframework/gitresources": "^0.1014.0",
```

Comment:
Is this right? Shouldn't it be ^0.1014.1?

Reply:
We depends on the prerelease version before. 
We just released 0.1014.0, so we are just removing the pre-release suffix